README.md

```diff
@@ -2,20 +2,21 @@
-|    1.0.1    | 3.1.x | 36.0 | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                                                                                                                                               <ul><li>Some resiliency added when VCD cells are restarted (34c1689)</li><li>Added Org ID to cert query so that `system/administrator` can also create https load balancers (44c72ab)</li></ul>                                                                                                                                                               |<ul><li>1.21</li><li>1.20</li><li>1.19</li></ul>|[CPI 1.0.x docs](https://github.com/vmware/cloud-provider-for-cloud-director/tree/1.0.x)|
-|    1.0.0    | 3.1.x | 36.0 | 10.3.1+ <br/>(10.3.1 needs hot-patch to prevent VCD cell crashes in multi-cell environments) |                                                                                                                                                                                                                           Needs NSX-T 3.1.1 with NSX Advanced Load Balancer (Avi) version 20.1.3+                                                                                                                                                                                                                           |<ul><li>1.21</li><li>1.20</li><li>1.19</li></ul>|[CPI 1.0.0 docs](https://github.com/vmware/cloud-provider-for-cloud-director/tree/1.0.0)|
+| CPI Version | CSE Version | VMware Cloud Director API | VMware Cloud Director Installation | Notes  | Kubernetes Versions | Docs |
+|:-----------:|:-----------:|:-------------------------:|:----------------------------------:|:------:|:-------------------:|:----:|
+|    1.6.0    | 4.2.0+ | 37.2 | 10.4.3+ | <ul><li>Added support for Gateways that use IP spaces</li><li>Upgraded VCD API version used by CPI from 36.0 to 37.2</li><li>Bump golang.org/x/crypto from 0.14.0 to 0.17.0</li></ul> | <ul><li>1.28</li><li>1.27</li><li>1.26</li><li>1.25</li><li>1.24</li><li>1.23</li><li>1.22</li></ul> | [CPI 1.6.z docs](https://github.com/vmware/cloud-provider-for-cloud-director/tree/1.6.z) |
```

Comment:
This has not been followed in the past, but could we point to the tag for the docs rather than the branch (1.6.z)